tests/get_producers_tests.cpp

```diff
@@ -61,8 +61,10 @@ BOOST_AUTO_TEST_CASE( get_producers_from_table) { try {
       const auto& row = results.rows[0].get_object();
       BOOST_REQUIRE(row.contains("owner"));
       BOOST_REQUIRE_EQUAL(row["owner"].as_string(), "producer1111");
+
       // check for producer_authority not present, since it is only set when the producer schedule is used, this verifies producers table was used
-      BOOST_REQUIRE(!row.contains("producer_authority"));
+      // Disabling this check as the producer_authority appears to always be set with current system contract
```

Comment:
SIDE NOTE: I think the team should decide on a specific string that we use for any commented out tests (I would assume we would delete the tests if we understand that they are because we changed the fundamental way things work)

Reply:
@bearcubsvet  or @dtaghavi  -- I'd appreciate if one of you could clarify in this case if my assertion was correct and, if so, we could fully delete these lines.

Reply:
consensus at last call was that this was still unclear and we'll leave the comments in for now until we can more confidently resolve.